DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh

```diff
@@ -286,6 +288,9 @@ fi
 
 # ad-hoc options for GPU reco workflow
 export CONFIG_EXTRA_PROCESS_o2_gpu_reco_workflow+=";GPU_global.dEdxDisableResidualGainMap=1;$VDRIFTPARAMOPTION;$TRACKTUNETPCINNER;"
+if [[ $ALIEN_JDL_LPMPRODUCTIONTYPE == "MC" ]]; then
+  export CONFIG_EXTRA_PROCESS_o2_gpu_reco_workflow+=";GPU_global.dEdxDisableResidualGain=1"
```

Comment:
@chiarazampolli , I think we should use both:
```suggestion
  export CONFIG_EXTRA_PROCESS_o2_gpu_reco_workflow+=";GPU_global.dEdxDisableResidualGain=1;GPU_global.dEdxDisableResidualGainMap=1"
```